cairo/tests/fixtures/runner.py

```diff
@@ -344,7 +344,6 @@ def _factory(entrypoint, *args, **kwargs):
         if add_output:
             final_output = serde.serialize_list(output_ptr)
 
-        cumulative_retdata_offsets = serde.get_offsets(return_data_types)
```

Comment:
variable missing

Reply:
it was a redefinition, same line few lines above